### DISCUSSION TITLE
Welcome to HyperCoast Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [opengeos/HyperCoast#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hyanqi** [comment]
![微信图片_20240514161014](https://github.com/opengeos/HyperCoast/assets/169759244/bd60bd91-61ac-4e07-9019-db87ff7598e7)
I would like to ask why this code cannot run in jupyter and no error is reported. I can't find resources on the Internet to solve the problem, so I want to come here to ask. Can you help me answer it? The source website is this https:// hypercoast.org/examples/pace/, I have run all the previous ones, but it just doesn’t work here.

**Comment by LZXLQ** [comment]
![QQ截图20240614223611](https://github.com/opengeos/HyperCoast/assets/165672552/42989520-73fd-4770-926f-cbe44eda5e4c)
Hello, I would like to ask why my images can be loaded but cannot be interactive. Are there any solutions?

**Reply by giswqs** [reply]
Not sure what you mean. It is likely your installation problem. Try it on Google Colab. 
https://colab.research.google.com/github/opengeos/HyperCoast/blob/main/docs/examples/pace.ipynb

**Reply by LZXLQ** [reply]
I used to be able to interactively check the spectral curve info of images in JupyterLab, but now I can't. Not sure what's going on.

**Reply by giswqs** [reply]
Create a new conda env to install packages.

**Reply by LZXLQ** [reply]
ok,thanks

**Comment by LZXLQ** [comment]
![QQ截图20240615171824](https://github.com/opengeos/HyperCoast/assets/165672552/b9b4676b-fee5-4388-8b98-126b9f571f23)

I tried setting up a new environment, but I still can't interact with it. I can't rotate that 3D cube like you showed me. I'm doing this in JupyterLab, so what's going wrong?

**Reply by giswqs** [reply]
See https://tutorial.pyvista.org/tutorial/00_jupyter/index.html. This is not really a hypercoast issue. You can use any pyvista examples directly. If the interactive functionality does not work, then it is a pyvista issue.

**Comment by onurkaraca87** [comment]
Hi Dr Wu

I have two quick questions regarding these datasets and would greatly appreciate your guidance.
1) As shown in the attached image, the PACE dataset does not consistently cover our study area. I am currently using Hypercost in Jupyter Notebook. Do you know if Hypercost applies any filters that might affect the dataset's coverage?

![Screenshot_5](https://github.com/user-attachments/assets/f3b1e996-364a-4145-945e-88b41c09b1bf)

2) Is it possible to export PACE data in GeoTIFF format for further analysis? 
Thank you in advance for your time and assistance. I look forward to your response.

**Reply by giswqs** [reply]
1. No.
2. Yes.

Please go through the relevent notebook examples on the website.
